service/mantle/fedex/FedexServices.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<services xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/service-definition-2.1.xsd">
+
+    <!-- Generic Service Request -->
+    <service verb="set" noun="Connection">
+        <in-parameters>
+            <parameter name="method"/>
+            <parameter name="path"/>
+            <parameter name="contentType"/>
+            <parameter name="requestMap" type="Map"/>
+            <parameter name="requestPath"/>
+            <parameter name="apiTokenValue"/>
+            <parameter name="shippingGatewayConfigId"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="responseMap"/>
```

Comment:
Its good to specify the type of parameter if not String, by default it is String type. But here it is expected to return a map.